app/lib/bulk_download_service.rb

```diff
@@ -13,7 +13,8 @@ class BulkDownloadService
   #   - +user+ (User) => User requesting download
   #   - +study_bucket_map+ => Map of study IDs to bucket names
   #   - +output_pathname_map+ => Map of study file IDs to output pathnames
-  #   - +tdr_files+ => Hash of tdr accessions to arrays of file information including name & url for each file
+  #   - +azul_files+ => Hash of Azul file summary objects
+  #   - +source+ => Origin of bulk download request (either search or study-based download)
```

Comment:
Which context a user searched, the scope of the search: "study" or "global"
```suggestion
  #   - +context+ => Context of bulk download request ("study" for single-study, or "global" for search across all studies)
```

Reply:
735d0f0